pkg/transforms/genericResourceConfig.go

```diff
@@ -126,8 +126,17 @@ var defaultTransformConfig = map[string]ResourceConfig{
 		properties: []ExtractProperty{
 			{Name: "ready", JSONPath: `{.status.conditions[?(@.type=='Ready')].status}`},
 			{Name: "status", JSONPath: `{.status.conditions[?(@.type=='Ready')].reason}`},
+			{Name: "phase",  JSONPath: `{.status.phase}`},
 			{Name: "indications", JSONPath: `{.status.indications}`}, // this is an array of strings - will collect array items separated by ;
-			{Name: "sourceVM", JSONPath: `{.spec.source.name}`},
+			{Name: "_sourceVM", JSONPath: `{.spec.source.name}`},
+		},
+	},
+	"VirtualMachineRestore.snapshot.kubevirt.io": {
+		properties: []ExtractProperty{
+			{Name: "ready", JSONPath: `{.status.conditions[?(@.type=='Ready')].status}`},
+			{Name: "status", JSONPath: `{.status.conditions[?(@.type=='Ready')].reason}`},
+			{Name: "lastRestoreTime", JSONPath: `{.status.restoreTime}`},
+			{Name: "_targetVM", JSONPath: `{.spec.target.name}`},
```

Comment:
So, let's match the CLI

1. Rename lastRestoreTime -> restoreTime
2. Rename _targetVM -> targetName
3. Add targetKind   (I wonder if this can be anything other than VirtualMachine)

<img width="980" alt="image" src="https://github.com/user-attachments/assets/4515c110-ab01-4c16-bd99-58bc119920f4" />